examples/patient-hub/src/main/ml-config/databases/final-database.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I recommend gitignoring the src/main/ml-config directory. All of these files will be generated when the app is initialized, along with the hub-internal-config directory.